Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -122,6 +122,7 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 	int32 NumSpawnZones;
 #if	STATS
 	// For stat profile
+	int32 CPUProfileInterval;
```

Comment:
```suggestion
#if STATS
	int32 CPUProfileInterval;
#endif
```
This is unused and unset if `STATS` is not set/defined